src/app/shared/components/template/components/toggle-bar/toggle-bar.ts

```diff
@@ -7,7 +8,10 @@ import {
 } from "src/app/shared/utils";
 
 interface IToggleParams {
-  /** TEMPLATE PARAMETER: "variant" */
+  /**
+   * TEMPLATE PARAMETER: "variant". Setting "ios" or "android" will style the toggle to match the respective
+   * platform, otherwise the default is to match the current device platform, using "ios" on web.
```

Comment:
@jfmcquade I think this code comment line disagrees with what you've said in [your comment on the PR](https://github.com/IDEMSInternational/open-app-builder/pull/2352#issuecomment-2305169855)(?): 

>If no variant is explicitly set by the author, the toggle will default to having the "android" appearance on Android and web, else on iOS it will have the "ios" appearance

So - on web is the default to use `ios` or `android`?

Reply:
On web the default is `android`. I made this default in response to Chris's [comment](https://github.com/IDEMSInternational/open-app-builder/pull/2352#discussion_r1725936151) that most of our user-base use Android, and so it probably makes sense for the web preview to match the styling that they will see.